### ISSUE TITLE
After `restore` to another machine user `dendrite` is not part of `ssl-cert` group

### ISSUE THREAD

**Issue by orhtej2** [YunoHost-Apps/dendrite_ynh#110]
### Describe the bug

After backing up `dendrite` and transferring archive to another machine I restored the app. Service `dendrite` however failed to start with

```
Aug 07 23:35:15 domain.tld dendrite[75360]: time="2023-08-07T21:35:15.144797224Z" level=info msg="Starting external listener on :8008"
Aug 07 23:35:15 domain.tld dendrite[75360]: time="2023-08-07T21:35:15.145984471Z" level=info msg="Starting external listener on :8449"
Aug 07 23:35:15 domain.tld dendrite[75360]: time="2023-08-07T21:35:15.146170567Z" level=fatal msg="failed to serve HTTPS" error="open /etc/yunohost/certs/domain.tld/crt.pem: permission denied"
Aug 07 23:35:15 domain.tld systemd[1]: dendrite.service: Main process exited, code=exited, status=1/FAILURE
Aug 07 23:35:15 domain.tld systemd[1]: dendrite.service: Failed with result 'exit-code'.
```

However, user was added to the group according to logs attached below, no errors:

```
2023-08-07 23:34:21,671: DEBUG - Adding user postgres to group ssl-cert
```

Manually running `sudo usermod -a -G ssl-cert dendrite` fixed the problem.

### Context

- Hardware: VPS bought online
- YunoHost version: 11.2.3
- I have access to my server: Through SSH
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
- Using, or trying to install package version/branch: 0.13.1

### Steps to reproduce

1. Back up dendrite
2. Transfer backup to another machine
3. Restore.

### Expected behavior

Dendrite works.

### Logs

https://paste.yunohost.org/raw/dininakosu

**Comment by orhtej2** [comment]
Wait, it added `postgres` not `dendrite` to `ssl-cert` oO

**Comment by tituspijean** [comment]
I think this line is triggered during the postinstall script of PostgreSQL. *However*, I encountered the same issue as you, and issued d00f1f68424bbb958b8c71e465b777b0100bc8af to fix it. That's very weird it did not fix it.

**Comment by orhtej2** [comment]
It's not in my backup and now I see your change is after last version bump in `manifest.toml` hence was not picked up on my end. 

Perhaps we should bump version to `0.13.1~ynh3`?

**Comment by orhtej2** [comment]
Addressed in #111
